### PR TITLE
Fix 14084: Removed Airflow Mssql backend connection

### PR DESCRIPTION
**Pull request by OnkarVO7**
### Describe your changes:

Fixes https://github.com/open-metadata/OpenMetadata/issues/14084

#
### Type of change:

- [ ] Bug fix
- [x] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: bootstrap/sql/migrations/native/1.3.0/mysql/schemaChanges.sql [hunk 1]
```diff
@@ -32,8 +32,14 @@ set json = JSON_INSERT(
 )
 where name = 'DataInsightsApplication';
 
+-- Remove mssql connection from airflow db
+UPDATE pipeline_service_entity pse 
+SET json = JSON_REMOVE(json, '$.connection.config.connection')
```
Comment (pmbrull) [inline bootstrap/sql/migrations/native/1.3.0/mysql/schemaChanges.sql:37]: will this break the service? Or do we need to specify a "Backend" connection meanwhile or something 🤔

**Review by pmbrull — CHANGES_REQUESTED** [review]
hi @OnkarVO7 thanks for tackling this. Let's update the 1.3.x docs to have this highlighted on the breaking changes section

File: bootstrap/sql/migrations/native/1.3.0/mysql/schemaChanges.sql (hunk 1 above)
Comment (OnkarVO7) [reply bootstrap/sql/migrations/native/1.3.0/mysql/schemaChanges.sql:37]: I checked it, it does not break the service. The connection shows blank in the UI and then the same can be updated via UI

**Comment by OnkarVO7** [comment]
> hi @OnkarVO7 thanks for tackling this. Let's update the 1.3.x docs to have this highlighted on the breaking changes section

@pmbrull added the breaking changes please take a look

**Review by pmbrull — APPROVED** [review]

**Review by ayush-shah — APPROVED** [review]

**Pull request merged**